src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java

```diff
@@ -31,13 +36,11 @@
 
   private static boolean doneInstallPlugin = false;
 
-  @BeforeClass
+//  @BeforeClass
```

Comment:
Great catch! I did this temporarily during testing and forgot to change it back.